src/nccl_ofi_api.c

```diff
@@ -452,6 +457,11 @@ ncclResult_t nccl_net_ofi_accept_v4(void* listenComm, void** recvComm)
 {
 	ncclResult_t ret = ncclInvalidArgument;
 
+	if (0 == strcasecmp(nccl_ofi_selected_protocol, "RDMA")) {
```

Comment:
Why wait until accept?  Why not abort in listen?

Reply:
For consistency, how about we just fail in all three?

Reply:
Done